mobile/search.go

```diff
@@ -6,17 +6,24 @@ import (
 	"github.com/textileio/textile-go/pb"
 )
 
+type CancelFn struct {
+	cancel *broadcast.Broadcaster
+	done   func()
+}
```

Comment:
Here's this `CancelFn`... internally `call` wrangles the cancel broadcaster. Those params don't make it across the bridge, but `call` does, so all good